apps/vue-showcase/cypress/e2e/vue-route-guard.cy.ts

```diff
@@ -0,0 +1,66 @@
+describe(`@this-dot/vue-route-guard`, () => {
+  describe(`No authentication set`, () => {
+    before(() => {
+      cy.window().then((win) => {
+        win.sessionStorage.removeItem('VUEROUTEGUARD-TOKEN');
+      });
+      cy.clearLocalStorage('VUEROUTEGUARD-TOKEN');
+      cy.clearCookie('VUEROUTEGUARD-TOKEN');
+    });
+
+    it(`redirect to login`, () => {
+      cy.visit('/route-guard');
+
+      cy.url().should('contain', 'route-guard/login');
+    });
+
+    it(`does not redirect to home view when home navigation is clicked`, () => {
+      cy.visit('/route-guard');
+
+      cy.get('#homeRoute').should('be.visible').click();
+      cy.url().should('not.eq', Cypress.config().baseUrl + '/route-guard');
+    });
+
+    it(`about navigation is not displayed`, () => {
+      cy.visit('/route-guard');
+
+      cy.get('#aboutRoute').should('not.exist');
```

Comment:
I usually recommend not using the `id` property for these elements. I always put `data-test-id` attributes on my elements in the app, and then use those to get the elements in the test.

Please see the other demoapp's e2e tests for examples. I'd like the demoapp tests to be consistent in this matter.